recipes/versioningit/meta.yaml

```
@@ -0,0 +1,68 @@
+{% set name = "versioningit" %}
+{% set version = "1.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3b7565a2db92f1206b5b833a72cc6936c8a74d65e94224f0a9eb54799d1561d9
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - versioningit = versioningit.__main__:main
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools
+  run:
+    - python >=3.6
+    - dataclasses
+    - importlib-metadata >=3.6
+    - packaging
+    - setuptools
+    - tomli >=1.2,<3.0
+
+test:
+  requires:
+    - build >=0.7.0
+    - git >=2.32
+    - mercurial >=5.2
+    - setuptools >=42
+    - pip
+    - pydantic >=1.8,<2.0
+    - pytest >=7.0
+    - pytest-cov >=3.0
+    - pytest-mock >=3.0
+    - wheel
+  source_files:
+    - test
+    - tox.ini
+  imports:
+    - versioningit
+  commands:
+    - python -m pytest -vv test
```

```suggestion
    - pip check
    - versioningit --help
    - python -m pytest -vv test
```
You *can* run your entire test suite, but we're more interested in testing whether the packaging was correct. For a pure-python package this means checking that entry-points and dependencies were installed correctly.